srcCpp/Infrastructure_micro.h

```diff
@@ -159,16 +159,59 @@ class NDDSUtility {
     #endif
     };
 
+    /* Get the time cost to call getTimeUsec a number of times equal to iteration */
+    static unsigned long long get_time_overhead(unsigned int iterations) {
+
+        PerftestClock clock = PerftestClock::getInstance();
+        unsigned long long init = 0, fin = 0, overhead = 0;
+
+        init = clock.getTimeUsec();
+        for (int i = 0; i < iterations; i++) {
+            clock.getTimeUsec();
+        }
+        fin = clock.getTimeUsec();
+
+        return fin - init;
```

Comment:
I believe overhead is not used, right? In that case it can be deleted. then you can just declare 

```
unsigned long long init = clock.getTimeUsec();
```

and then:

```
return clock.getTimeUsec() - init;
```